src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -584,6 +585,8 @@ public void It_publishes_on_release_if_PublishRelease_property_set_in_csproj()
         [InlineData("-property:Configuration=Debug")]
         [InlineData("--property:Configuration=Debug")]
         [InlineData("/p:Configuration=Debug")]
+        [InlineData("-p:_IsPublishing=true;Configuration=Debug")]
```

Comment:
this test used to have the surrounding ' but caused unspeakable disaster (it seems to work but not under test...) 
per @/baronfel
https://nullprogram.com/blog/2022/02/18/
It's the wild west of Windows command line parsing 😬